### ISSUE TITLE
Rename to clinical-meteor

### ISSUE THREAD

**Issue by awatson1978** [clinical-meteor/cookbook#89]

**Comment by alimgafar** [comment]
Hi Abigail,

The other day, at DevShop NY, I mentioned your cookbook as a resource to the new members. Now, I haven't been on the repo in a few months so I was surprised to see it changed to clinical-meteor. Of course, Adrian and I looked a little puzzled, but he smoothly pointed out some other resource and moved on. 

Is the move to make the cookbook exclusively for the clinical domain? Or will it continue to have general applicability? I'm curious to know where you're taking it.

Best wishes,

Alim

**Comment by awatson1978** [comment]
Hi Alim,
The cookbook is still 100% applicable to general Meteor; but it's no longer
tracking bleeding-edge development and keeping up with the entire
community.  Clinical Track is designed for anybody in healthcare or the
life sciences who needs a stable Meteor distro with a validation &
verification QA framework.

I got tired of MDG breaking the build every few months and failing to
deliver a QA system, so I went and implemented one myself using libraries
that I knew could pass a V&V inspection.  So, I'm not entirely sure how to
answer your question.  Is CentOS exclusively for the server domain because
it ships with datacenter management utilities?  Is Ubuntu exclusively for
the desktop domain because it ships with music players and games?  Or do
they each have general applicability?
Best,
Abigial

ps.  Looks like I may be back in NYC this summer.

On Thu, Mar 17, 2016 at 9:54 PM, Alim S. Gafar <notifications@github.com>
wrote:

> Hi Abigail,
>
> The other day, at DevShop NY, I mentioned your cookbook as a resource to
> the new members. Now, I haven't been on the repo in a few months so I was
> surprised to see it changed to clinical-meteor. Of course, Adrian and I
> looked a little puzzled, but he smoothly pointed out some other resource
> and moved on.
>
> Is the move to make the cookbook exclusively for the clinical domain? Or
> will it continue to have general applicability? I'm curious to know where
> you're taking it.
>
> Best wishes,
>
> Alim
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/clinical-meteor/cookbook/issues/89#issuecomment-198206796>
>



-- 
Abigail Watson
Clinical Applications Architect
Pentasyllabic Studios, LLC
343 Soquel Ave #529
Santa Cruz, CA 95062
awatson1978@gmail.com
347-302-9345 (mobile)
awatson1978 (skype)
